src/ports/biconomy/types.ts

```diff
@@ -0,0 +1,65 @@
+import { ErrorCode } from 'decentraland-transactions'
+import {
+  IMetaTransactionProviderComponent,
+  TransactionData,
+} from '../../types/transactions/transactions'
+
+export type MetaTransactionRequest = TransactionData & {
+  apiId: string
+}
+
+export type BiconomyMetaTransactionComponent = IMetaTransactionProviderComponent
+export type MetaTransactionResponse = {
+  txHash?: string
```

Comment:
here's what I meant from my other comment, if we can remove this `?` and so remove the `!` from the other file.